source/index.html.md

```diff
@@ -18,91 +18,121 @@ search: true
 Welcome to the Kontist API. Kontist allows you to access your own bank transaction data as well as the initiation and authentication of financial transactions. Please note, that you will still need to confirm the latter with an individual authorization code which you will receive through SMS.
 
 # Authentication
+To manage the data via our API your application needs to gain access on behalf of the user. This is done through obtaining an access token via [OAuth2](https://tools.ietf.org/html/rfc6749). The access token must then be send in each request in the HTTP header like this: "Authorization: Bearer TOKEN".
 
-You can start playing with the Kontist API using [HTTP Basic Auth](https://en.wikipedia.org/wiki/Basic_access_authentication). However this is not to be used in production as it is highly rate limited. The documentation of our oAuth flow is in preparation and will be added at a later point in time.
+You need two kinds of credentials to get such a token: The first part is a fixed pair of client id and client secret. They identify your client application which connects to the API. Each application has its own pair of client id and secret, please contact us via developer@kontist.com to get your own client credentials.
+The second part is obtained through the user and can be done in several ways, here we describe the preferred way through the "Authorization Code" grant type.
 
-`Authorization: Basic QWxhZGRpbjpPcGVuU2VzYW1l`
+## Authorization Code
+In general, the process looks like this:
+1. You redirect the user in a browser to an url on our end.
+2. The user is required to login and needs to accept your applications authorization request. The browser redirects back to your application with a `code` parameter.
```

Comment:
Nit: Shouldn't it be `your application's authorization request` with a `'` ?

Reply:
Sure, done.